middleware/errorHandlers.js

```diff
@@ -0,0 +1,13 @@
+exports.sqlErrors = (err, req, res, next) => {
+  next(err)
+}
+
+exports.customErrors = (err, req, res, next) => {
+  if(err.errCode) {
+    return res.status(err.errCode).send({msg: err.errMsg})
+  } else {
+    console.log(err)
+    return res.status(500).send({msg: "Internal server error"})
+  }
+
+}

```

Comment:
This should really be 2 different error handlers and we should invoke next to move onto the 500 server error handler.